src/Validation/Validation.php

```diff
@@ -1332,7 +1332,11 @@ public static function uploadError($check, bool $allowNoFile = false): bool
     {
         if ($check instanceof UploadedFileInterface) {
             $code = $check->getError();
-        } elseif (is_array($check) && isset($check['error'])) {
+        } elseif (is_array($check)) {
+            if (!isset($check['error'])) {
+                return false;
```

Comment:
Makes sense, but codecov says there isn't a test that covers no errors?

Reply:
`error` key not being set would mean an invalid file array. I don't really want to bother with adding a test for this since we have already switched to using `UploadedFileInterface` instances by default.